azure-pipelines.yml

```diff
@@ -0,0 +1,37 @@
+trigger:
+- master
```

Comment:
Maybe remove this so feature branches are tested too?

Reply:
Sure can. I rely on travis for that noise on my projects.